patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -168,7 +193,26 @@
                  {
                      d0 = MathHelper.func_151237_a(d0 * 8.0D, worldserver1.func_175723_af().func_177726_b() + 16.0D, worldserver1.func_175723_af().func_177728_d() - 16.0D);
                      d1 = MathHelper.func_151237_a(d1 * 8.0D, worldserver1.func_175723_af().func_177736_c() + 16.0D, worldserver1.func_175723_af().func_177733_e() - 16.0D);
-@@ -2604,7 +2632,7 @@
+@@ -2554,8 +2589,7 @@
+                 d1 = (double)MathHelper.func_76125_a((int)d1, -29999872, 29999872);
+                 float f = this.field_70177_z;
+                 this.func_70012_b(d0, this.field_70163_u, d1, 90.0F, 0.0F);
+-                Teleporter teleporter = worldserver1.func_85176_s();
```

Comment:
This is inside an `if (false` patched block, is it not?

Reply:
It's outside the patched-out blocks. The dummied out code is just above, and adjusts `d0` and `d1` based on the precoded scale factor of `8.0` for Nether travel.

Reply:
ahh my bad